docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+phpfpm:
+  command: php-fpm --allow-to-run-as-root
+  image: php:5.6.8-fpm
+  volumes:
+    - fpm.conf:/usr/local/etc/php-fpm.conf
```

Comment:
@alexanderilyln I got `file exists at /mnt/sda1/var/lib/docker/aufs/mnt/eb13125c1167ab4a391994d402ac8181b13901875d2c1350793452242ca275b2/usr/local/etc/php-fpm.conf, can't create volume there` error when `docker-compose up`. Same to other NGINX config below.

I am using boot2docker, am I missing anything?